.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: build
```

Comment:
The name could be "Tests" which is more specific for what it does

Reply:
Yes. It's right. I will change into `Tests` name.